File: 20.py
# 648
# TODO: find a way to make this in C!
def factorial(n):
    if n == 0 or n == 1:
        return 1
    else:
        return n*factorial(n - 1)

n = factorial(100)
sum_digits = 0

while n != 0:
    sum_digits += n%10
    n //= 10

print(sum_digits)

File: 20.c
// 648
#include <stdio.h>

long long int factorial(long long int n)
{
    if (n == 0 || n == 1)
        return 1;
    else
        return n*factorial(n - 1);
}

int main()
{
    long long int n = factorial(21);
    printf("%lld\n", n);
    return 0;
}
